bobber/lib/exit_codes.py

```diff
@@ -1,6 +1,10 @@
 # SPDX-License-Identifier: MIT
 # This file contains a list of exit codes for debugging
 SUCCESS = 0  # Successful termination
-BASELINE_FAILURE = -10  # Performance did not meet criteria
-MISSING_LOG_FILES = -20  # Parsing directory with no logs
-DOCKER_BUILD_FAILURE = -30  # Failure building Docker image
+BASELINE_FAILURE = 10  # Performance did not meet criteria
```

Comment:
So we switched back to positive numbers here? Any particular reason?

Reply:
Correct. I put the explanation in the commit message and the PR, but basically Python doesn't handle negative exit codes and throws a semi-random number instead, so by switching back to positive numbers we get the actual exit values we expect.

Reply:
My mistake - thanks for the clarity!

Reply:
No worries! Great catch!